lib/modules/manager/regex/readme.md

```diff
@@ -1,24 +1,26 @@
-The `regex` manager is designed to allow users to manually configure Renovate for how to find dependencies that aren't detected by the built-in package managers.
+With the `regex` manager you can configure Renovate so it finds dependencies that are not detected by its built-in package managers.
 
-This manager is unique in Renovate in that:
+The `regex` manager is unique in Renovate because:
 
 - It is configurable via regex named capture groups
-- Through the use of the `regexManagers` config, multiple "regex managers" can be created for the same repository
 - It can extract any `datasource`
+- By using the `regexManagers` config, you can create multiple "regex managers" for the same repository
 
-We have [additional Handlebars helpers](https://docs.renovatebot.com/templates/#additional-handlebars-helpers) which help you perform common transformations on the regex manager's template fields.
+We have [additional Handlebars helpers](https://docs.renovatebot.com/templates/#additional-handlebars-helpers) to help you perform common transformations on the regex manager's template fields.
 Also read the documentation for the [`regexManagers` config option](https://docs.renovatebot.com/configuration-options/#regexmanagers).
 
 ### Required Fields
 
-The first two required fields are `fileMatch` and `matchStrings`.
-`fileMatch` works the same as any manager, while `matchStrings` is a `regexManagers` concept and is used for configuring a regular expression with named capture groups.
+The first two required fields are `fileMatch` and `matchStrings`:
 
-In order for Renovate to look up a dependency and decide about updates, it then needs the following information about each dependency:
+- `fileMatch` works the same as any manager
+- `matchStrings` is a `regexManagers` concept and is used for configuring a regular expression with named capture groups
+
+Before Renovate can look up a dependency and decide about updates, it needs this information about each dependency:
 
 - The dependency's name
-- Which `datasource` to look up (e.g. npm, Docker, GitHub tags, etc)
-- Which version scheme to apply (defaults to `semver`, but also may be other values like `pep440`)
+- Which `datasource` to use: npm, Docker, GitHub tags, and so on
```

Comment:
```suggestion
- Which `datasource` to use: npm, Docker, GitHub tags, and so on. For how to format this references see [datasource overview](https://docs.renovatebot.com/modules/datasource/#supported-datasources)
```
I think we should link to the datasource overview, else we will see people trying to use `Docker` or `Github tags` instead of `docker` and `github-tags`.